README.md

```diff
@@ -1 +1,24 @@
-# ckanext-custom-schema
\ No newline at end of file
+# ckanext-custom_schema
+
+A CKAN extension to customize datasets' schema. By adding custom fields to the schema, this extension allows user to provide more information about datasets.
+
+This extension adds customized metadata fields to datasets. After enableing it, go to ${URL}/dataset/edit/${YOUR-DATASET} to edit the custom fields.
+
+# Repo Structure 
+This is a mono-repo. Each branch stands for a customer. Before doing any work, please make sure that you are on the branch. 
+
+# Dependencies
+This extension is dependent on the `ckanext-scheming` extension. Before install this extension, click [here](https://github.com/OpenGov-OpenData/ckanext-custom-schema) to install the `ckanext-scheming` extension.
```

Comment:
The link goes to https://github.com/OpenGov-OpenData/ckanext-custom-schema instead of https://github.com/OpenGov-OpenData/ckanext-scheming